contribution.md

```diff
@@ -0,0 +1,54 @@
+
+*A Note to Developers* When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change. This  way, we can ensure that there is no overlap between contributions and internal development work. You may contact us at frc@autodesk.com.
+
+When ready to contribute, just submit a pull request and be sure to include a clear and detailed description of the changes you've made so that we can verify them and eventually merge.
+
+### Participate in a Code & Creativity Challenges or Contest
+Join a Synthesis code challenge or contest. Follow us @synthesis.adsk and @synthesis_adsk on social media or check out Chief Delphi to stay up to date with our latest activity.
+
+### Submit a CAD Model
+Submit your team's CAD model to be added to the Synthesis robot and field libraries by emailing your designs to frc@autodesk.com.
+
+### Write Tutorials, Increase Documentation
+We are always interested in ways to make our tutorials and documentation more clear to our end users. If there is content missing or could be refined, please follow our [contribution guidelines](#How-to-Contribute) to submitting a change.
+
+### Translate our Tutorials and Documentation
+If you or someone you know can speak and write in another language, we would like to translate our text-based resources to make available in more languages. Contact frc@autodesk.com for more details.
+
+### Create How-To or Project DIY and Inspiration Guides
+Did you add a feature to Synthesis? Learn how to use a specific feature? Write a how-to guide or share your project with us frc@autodesk.com and [@synthesis.adsk](https://www.instagram.com/synthesis.adsk/). 
+
+### Share a Use Case Story
+Hearing how you use Synthesis is valuable feedback to our team. Share your stories by tagging us [@synthesis.adsk](https://www.instagram.com/synthesis.adsk/) or [@synthesis_adsk](https://twitter.com/synthesis_adsk) or send them to frc@autodesk.com with a reference to this guide.
+
+### Module Development Utilzing the Synthesis API
+With the 2020 release, Synthesis now provides a [public API](https://www.nuget.org/packages/Autodesk.Synthesis.Module.API/) in which developers can use to write their own modules. Creating your own modules allows you to customize your experience by adding features or expanding existing features, like expanding programming languages.
+
+**Expand Programming Languages**
+Currently, Synthesis supports C++ and Python code emulation. We recognize that teams use different languages for their robot code so being able to expand our emulation support is one of our goals. Learn more about contributing development here. 
+
+**Expand FIRST Support** 
+FIRST control systems and essentials like sensors, cameras, various motors, etc. would greatly increase simulation support. Learn more about contributing development here. 
+
```

Comment:
Perhaps also mention that people can submit a PR to contribute to existing modules or new modules if they believe it would be beneficial to ship with the official release. We'd have to evaluate if it's worthy of course.